spec/csv_importer_spec.rb

```diff
@@ -134,6 +134,44 @@ class ImportUserCSVByFirstName
         "confirmed_at" => Time.new(2012)
       )
     end
+
+    it 'imports even if there are comment lines' do
+      csv_content = "#this is technically not allowed in CSV but ... users!\nemail,confirmed,first_name,last_name
+BOB@example.com,true,bob,,"
+
+      import = ImportUserCSV.new(content: csv_content)
+
+      expect(import.valid_header?).to be_truthy
+
+      expect(import.rows.size).to eq(1)
+
+      row = import.rows.first
+
+      expect(row.csv_attributes).to eq(
+        {
+          "email" => "BOB@example.com",
+          "first_name" => "bob",
+          "last_name" => nil,
+          "confirmed" => "true"
+        }
+      )
+
+      import.run!
+
+      expect(import.report.valid_rows.size).to eq(1)
+      expect(import.report.created_rows.size).to eq(1)
+
+      expect(import.report.message).to eq "Import completed: 1 created"
+
+      model = import.report.valid_rows.first.model
+      expect(model).to be_persisted
+      expect(model).to have_attributes(
+        "email" => "bob@example.com", # was downcased!
+        "f_name" => "bob",
+        "l_name" => nil,
+        "confirmed_at" => Time.new(2012)
```

Comment:
Do not use `Time.new` without zone. Use one of `Time.zone.local`, `Time.new.current`, `Time.new.in_time_zone`, `Time.new.utc`, `Time.new.getlocal`, `Time.new.iso8601`, `Time.new.jisx0301`, `Time.new.rfc3339`, `Time.new.to_i`, `Time.new.to_f` instead.